src/QtLocationPlugin/QGCTileCacheWorker.cpp

```diff
@@ -97,9 +97,9 @@ QGCCacheWorker::enqueueTask(QGCMapTask* task)
         task->deleteLater();
         return false;
     }
```

Comment:
This check was prone to CPU cache not getting updated.

![QGCCacheWorker _valid is racey](https://user-images.githubusercontent.com/72767521/114422714-255c7b80-9b7c-11eb-8768-95763545b75c.png)